app/decorators/notification_decorator.rb

```diff
@@ -30,4 +32,103 @@ def milestone_count
 
     action.split("::").third
   end
+
+  def siblings
+    @siblings ||= begin
+      aggregated_data = json_data.dig("reaction", "aggregated_siblings")
+      aggregated_data ||= []
+      aggregated_data.select { |n| n["created_at"] > 24.hours.ago }
+    end
+  end
+
+  def to_model
+    self
+  end
+
+  def to_partial_path
+    return "notifications/#{notifiable_type.downcase}" unless reaction?
```

Comment:
So, there's a lot of changes to `NotificationDecorator`, but this is the key change: I wanted to find a way to introduce the new, multiple reactions view code (behind a feature flag), and the way we were rendering notifications was hard to unravel (sometimes `aggregate`, sometimes `single`, sometimes `article` or `comment` but then _actually_ `aggregate`...). I tried to move the old code to one side (`original` file prefix), but tried to find a less-confusing way to specify which notification partial is being rendered. 

I do think this makes the decorator is a bit more complicated than I'd like, but I'd rather have that than have an overly-complicated view (or, in this case: views), so it felt like the right trade-off to me.

Reply:
I attempted to [write a temporary comment here](https://github.com/forem/forem/pull/19020/commits/63beeec7fbef086fcee4226f5d0ed2bedc2c4bfb).

Reply:
Love this refactor!